api/src/test/java/org/openmrs/util/ConceptReferenceRangeUtilityTest.java

```diff
@@ -0,0 +1,326 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.util;
+
+import org.apache.velocity.exception.ParseErrorException;
+import org.joda.time.DateTimeUtils;
+import org.joda.time.LocalTime;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.openmrs.Concept;
+import org.openmrs.ConceptDatatype;
+import org.openmrs.Encounter;
+import org.openmrs.Location;
+import org.openmrs.Obs;
+import org.openmrs.Person;
+import org.openmrs.api.ConceptService;
+import org.openmrs.api.ObsService;
+import org.openmrs.test.jupiter.BaseContextSensitiveTest;
+
+import java.util.Calendar;
+import java.util.Collections;
+import java.util.Date;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+
+class ConceptReferenceRangeUtilityTest extends BaseContextSensitiveTest {
+	private Calendar calendar;
+	private Person person;
+	private ConceptReferenceRangeUtility conceptReferenceRangeUtility;
+	
+	@Mock
+	private ObsService obsService;
+	
+	@Mock
+	private ConceptService conceptService;
+	
+	@BeforeEach
+	public void setUp() {
+		person = new Person();
+		conceptReferenceRangeUtility = new ConceptReferenceRangeUtility();
+	}
+
+	@Test
+	public void testAgeInRange_shouldReturnTrueIfAgeIsWithinRange() {
+		calendar = Calendar.getInstance();
+		calendar.add(Calendar.YEAR, -5);
+		person.setBirthdate(calendar.getTime());
+		final String CRITERIA = "$patient.getAge() > 1 && $patient.getAge() < 10";
```

Comment:
If you use the criteria constant once, why not just directly pass it into the method?